extension/controller/backgroundScript.js

```diff
@@ -326,6 +343,24 @@ const messageListener = function(message, sender) {
             translatingAsBrowse: message.translatingAsBrowse
           });
           break;
+        case "errorCollectionConsent":
+          browser.storage.local.set({ errorCollectionConsent: message.consent });
+          Sentry.getCurrentHub().getClient()
+            .getOptions().enabled = message.consent;
+          for (const [key,] of telemetryByTab) {
```

Comment:
it seems we don't need to send this to tabs anymore, sentry is present only in bgScript